src/ui/components/UploadScreen/Sharing.tsx

```diff
@@ -36,11 +51,17 @@ function EditableSettings({
         <TeamSelect {...{ workspaces, handleWorkspaceSelect, selectedWorkspaceId }} />
       ) : null}
       <div
-        className="space-x-2 select-none flex flex-row items-center justify-between w-full border border-textFieldBorder rounded-md shadow-sm px-2.5 py-1.5 text-left cursor-default focus:outline-none focus:ring-1 focus:ring-primaryAccent focus:border-primaryAccentHover bg-jellyfish"
-        onClick={() => setIsPublic(!isPublic)}
+        className={classNames(
+          publicDisabled ? "opacity-60" : undefined,
```

Comment:
i like your logic with disabling if the user has multiple teams. if the has one org, then it's preferred to hide it.

Reply:
I took a slightly different approach for this. We could totally do precisely this but it requires a bit more clean up (e.g. if you remove the public toggle, the team select should go full width but the dropdown is configured for a fixed width). 

Instead, I figure the 80% case for orgs is that a user is only part of 1 workspace (the org) and both the library and public recordings are disabled so we can skip the edit button altogether and just report that the recording will land in the org and be private.